README.md

```diff
@@ -29,7 +29,7 @@ jobs:
 
         # This is needed to link the correct project.
         # Get it via: symfony project:info command
-        SYMFONY_PROJECT_ID: ${{ secrets.SYMFONY_PROJECT_ID }}
+        SYMFONY_PROJECT: xxxxxxxxxxxxx
```

Comment:
this is not considered a sensible value

Reply:
@tucksaun 

Why remove the `_ID` suffix?

Running

```
$ symfony link --help
```

yields

``` 
Usage:
  symfony project:link [options] [--] [<id>]

Arguments:
  id

Options:
  --force, -f  Override any previous link [default: false]

```

- so I believe the `_ID` suffix makes sense here.

What do you think?